web/controllers/category_controller.ex

```diff
@@ -4,10 +4,11 @@ defmodule CodeCorps.CategoryController do
   alias CodeCorps.Category
   alias JaSerializer.Params
 
+  plug :load_resource, model: Category, only: [:show]
```

Comment:
Pretty much. It's provided by `Pundit`, same as `load_and_authorize_resource`. It loads by `id` by default (we can provide a name for the `id` key), and assigns it to `conn.assings.underscored_model_name`. `load_and_authorize_resource` does the exact same thing but then also performs authorization on it. Also, if the resource with that id is not found, it automatically handles the error using `handle_not_found`, which we defined in `authentication_helpers`. That handler renders a JSON 404 error, unlike the default handler which triggers when we manually try and fail to load a resource, which renders an HTML 404 error.